### PR TITLE
Docs for Readme

### PR DESCRIPTION
**Pull request by mariehposa**
#### What does this PR do?
It provides project details.
#### Description of Task to be completed?
Breaking down into end to end test and contribution details
#### How should this be manually tested?
Check the readme file.
#### Any background context you want to provide?
None
- [*] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,77 +2,55 @@
...
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Authors
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+* **Pascal Ulor** - *Team lead* - [github](https://github.com/PascalUlor)
+* **Mariam Adedeji** - *Team member* - [github](https://github.com/mariehposa)
+* **Mike Perry Attara** - *Team member* - [github](https://github.com/mikeattara)
+* **Justinas Vebra** - *Team member* - [github](https://github.com/vebradev)
+* **Mildred Otieno** - *Team member* - [github](https://github.com/)
```
Comment (awuorm) [inline README.md]: @mariehposa my github username is awuorm, it's missing xD

Comment (attaradev) [inline README.md]:
```suggestion
* **Mildred Otieno** - *Team member* - [github](https://github.com/awuorm)
```

**Review by attaradev — COMMENTED** [review]
Look good to me.

**Review by attaradev — APPROVED** [review]

**Review by vebradev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mariehposa) [reply README.md]: I'm sorry about that, was a mistake. It has been updated by mike.

**Pull request merged**